images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -11,38 +11,78 @@ ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     PYTHONIOENCODING="UTF-8" \
     LANG="C.UTF-8" \
     LANGUAGE="C.UTF-8" \
-    LC_ALL="C.UTF-8"
-
-ARG IMAGES_DIR="/opt/tmp/images"
-RUN oryx prep --skip-detection --platforms-and-versions nodejs=14 --debug \
-    && echo "$DEBIAN_FLAVOR" \
-    && . /tmp/build/__goVersions.sh \
-    && downloadedFileName="go${GO_VERSION}.linux-amd64.tar.gz" \
-    && ${IMAGES_DIR}/retry.sh "curl -SLsO https://golang.org/dl/$downloadedFileName" \
-    && mkdir -p /usr/local \
-    && gzip -d $downloadedFileName \
-    && tar -xf "go${GO_VERSION}.linux-amd64.tar" -C /usr/local \
-    && rm -rf $downloadedFileName
-
-RUN set -ex \
-    # Install Python SDKs
-    # Upgrade system python
-    && PYTHONIOENCODING="UTF-8" \
-    # It's not clear whether these are needed at runtime...
-    && apt-get update \
+    LC_ALL="C.UTF-8" \
+    ORYX_PATHS="/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin"
+    
+RUN apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
+        git \
+        make \
+        unzip \
+        # The tools in this package are used when installing packages for Python
         build-essential \
+        moreutils \
         python3-pip \
         swig \
         tk-dev \
+        unixodbc-dev \
         uuid-dev \
-    && rm -rf /var/lib/apt/lists/*
+        # Required for PostgreSQL
+        libpq-dev \
+        # Required for mysqlclient
+        default-libmysqlclient-dev \
+        # Required for ts
+        moreutils \
+        rsync \
+        zip \
+        tk-dev \
+        uuid-dev \
+        #TODO : Add these to fix php failures. Check if these can be removed.
+        libargon2-0 \
```

Comment:
Added these packages to fix validation pipelines errors even though these are dynamically installed for PHP.

Reply:
Could you please clarify the consequences of pre-baking these? Should we have any concerns in terms of the size of adding these, or is it negligible and not worth looking into another solution to fix the validation pipeline?

Reply:
In terms of size, pre-baking these are pretty insignificant.

Reply:
Sounds good, thanks for clarifying